### PR TITLE
[release] 20230928

### PR DESCRIPTION
**Pull request by stwiname**
Release for CLI can come after we have updated the other SDKs and the common-* dependencies in the cli

### CODE REVIEW

File: packages/types-core/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@subql/types-core",
-  "version": "0.0.0-2",
+  "version": "0.0.0",
```
Comment (jiqiang90) [inline packages/types-core/package.json:3]: I wonder do we need a changelog for init release?

**Comment by jiqiang90** [comment]
I think we missed changelog for ts manifest ?

**Review by jiqiang90 — CHANGES_REQUESTED** [review]

File: packages/types-core/package.json (hunk 1 above)
Comment (stwiname) [reply packages/types-core/package.json:3]: I don't think so, there needs to be a reference point from what to base changes off.

**Comment by stwiname** [comment]
> I think we missed changelog for ts manifest ?
> 
> Will come after Cli release?

Ah was there no changelogs in the TS Manifest pr? Could you add changes to this PR?

**Review by jiqiang90 — APPROVED** [review]

**Pull request merged**